app/controllers/idv/verify_errors_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Idv
+  class VerifyErrorsController < ApplicationController
+    before_action :confirm_two_factor_authenticated
```

Comment:
I wonder if it would make sense to combine this controller and setup_errors_controller, since they're both really small and show related views, like the session_errors_controller that shows several different views.